android/assets/jsons/translations/template.properties

```diff
@@ -1232,7 +1232,7 @@ Annex =
 Annexed cities become part of your regular empire. = 
 Their citizens generate 2x the unhappiness, unless you build a courthouse. = 
 Puppet = 
-Puppeted cities do not increase your tech or policy cost, but their citizens generate 1.5x the regular unhappiness. = 
+Puppeted cities do not increase your tech or policy cost, but their citizens generate the regular unhappiness. = 
```

Comment:
No need for " but their citizens generate the regular unhappiness", we can just remove that

Reply:
Done. Sorry for the multiple commits, but I'm lazy to clone Unciv just for that.^^
Should I remove this part from the tutorial too? I think it useful for newcomers to know that puppet a city is still heavy on happiness, if not planned before.